packager/buildspec.yaml

```
@@ -54,6 +54,13 @@ foss:
         - rm plugin_*.go || true
         - GOOS=linux GOARCH=amd64 go generate --run plugin
 
+    aarch64_linux:
+      os: linux
+      arch: arm64
+      pre:
+        - rm plugin_*.go || true
+        - GOOS=linux GOARCH=amd64 go generate --run plugin
```

Or maybe (I should install go on this machine te tyr):

```suggestion
        - GOOS=linux GOARCH=arm64 go generate --run plugin
```